doc/classes/AudioStream.xml

```diff
@@ -39,10 +49,10 @@
 				Returns the length of the audio stream in seconds.
 			</description>
 		</method>
-		<method name="instance_playback">
+		<method name="instantiate_playback">
 			<return type="AudioStreamPlayback" />
 			<description>
-				Returns an AudioStreamPlayback. Useful for when you want to extend `_instance_playback` but call `instance_playback` from an internally held AudioStream subresource. An example of this can be found in the source files for `AudioStreamRandomPitch::instance_playback`.
+				Returns an AudioStreamPlayback. Useful for when you want to extend [method _instance_playback] but call [method instantiate_playback] from an internally held AudioStream subresource. An example of this can be found in the source files for [code]AudioStreamRandomPitch::instantiate_playback[/code].
```

Comment:
```suggestion
				Returns an AudioStreamPlayback. Useful for when you want to extend [method _instantiate_playback] but call [method instantiate_playback] from an internally held AudioStream subresource. An example of this can be found in the source files for [code]AudioStreamRandomPitch::instantiate_playback[/code].
```